src/test/resources/2502265.xml

```diff
@@ -42,28 +42,28 @@
             <title>Yahoo! Weather</title>
             <width>142</width>
             <height>18</height>
-            <link>http://weather.yahoo.com</link>
-            <url>http://l.yimg.com/a/i/brand/purplelogo//uh/us/news-wea.gif</url>
+            <link>https://weather.yahoo.com</link>
+            <url>https://l.yimg.com/a/i/brand/purplelogo//uh/us/news-wea.gif</url>
         </image>
```

Comment:
Test updated URLs. The above does not resolve.

Reply:
The image is the old logo, it didn't get migrated to the new server.
The new Yahoo weather icon is https://s.yimg.com/ge/new/WEATHER_ICON.png